src/popover/popover.directive.ts

```diff
@@ -199,20 +218,23 @@ export class PopoverDirective implements OnInit, OnDestroy {
     } else {
       showPopover();
     }
+    this.setAriaDescribedBy();
+    this.isOpen = true;
   }
 
   /**
    * Closes an element’s popover. This is considered a “manual” triggering of
    * the popover.
    */
-  hide(): void {
+  hide(): void [
```

Comment:
:thinking:  :thinking:  :thinking:  :thinking:   
I think, that definetely should be `{` here